src/main/java/org/folio/circulation/item/controller/TenantController.java

```diff
@@ -0,0 +1,59 @@
+package org.folio.circulation.item.controller;
+
+import jakarta.validation.Valid;
+import liquibase.exception.LiquibaseException;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import lombok.extern.log4j.Log4j2;
+import org.folio.spring.FolioExecutionContext;
+import org.folio.spring.liquibase.FolioSpringLiquibase;
+import org.folio.spring.service.TenantService;
+import org.folio.tenant.domain.dto.TenantAttributes;
+import org.folio.tenant.rest.resource.TenantApi;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@Log4j2
+@RestController("folioTenantController")
+@RequestMapping
+@RequiredArgsConstructor
+public class TenantController implements TenantApi {
```

Comment:
What will happen if we don't have this class?

Reply:
this is useless. correct.
I removed it.